zerver/openapi/zulip.yaml

```diff
@@ -5007,6 +5007,7 @@ paths:
         This API corresponds to the
         [delete a message completely][delete-completely] feature documented in
         the Zulip Help Center.
+      x-admin-config: true
```

Comment:
Let's call this `x-requires-administrator: true`; I don't think `admin-config` is going to read particularly intuitively.